FreeType.xs

```diff
@@ -628,7 +628,7 @@ qefft2_face_fixed_sizes (Font_FreeType_Face face)
         int i;
         FT_Bitmap_Size *size;
         HV *hash;
-        double pt, ppem;
+        double pt = 12.0, ppem;
```

Comment:
No. It is possible for size=null or size->size = 0 . The former will crash, the latter will bypass this section. Hopefully code before calling this knows better.